Mathlib/Analysis/NormedSpace/Multilinear/Basic.lean

```diff
@@ -362,43 +367,43 @@ theorem bounds_bddBelow {f : ContinuousMultilinearMap 𝕜 E G} :
     BddBelow { c | 0 ≤ c ∧ ∀ m, ‖f m‖ ≤ c * ∏ i, ‖m i‖ } :=
   ⟨0, fun _ ⟨hn, _⟩ => hn⟩
 
-theorem isLeast_opNorm : IsLeast {c : ℝ | 0 ≤ c ∧ ∀ m, ‖f m‖ ≤ c * ∏ i, ‖m i‖} ‖f‖ := by
+theorem isLeast_opNorm (f : ContinuousMultilinearMap 𝕜 E G) :
+    IsLeast {c : ℝ | 0 ≤ c ∧ ∀ m, ‖f m‖ ≤ c * ∏ i, ‖m i‖} ‖f‖ := by
   refine IsClosed.isLeast_csInf ?_ bounds_nonempty bounds_bddBelow
   simp only [Set.setOf_and, Set.setOf_forall]
   exact isClosed_Ici.inter (isClosed_iInter fun m ↦
     isClosed_le continuous_const (continuous_id.mul continuous_const))
 
 @[deprecated (since := "2024-02-02")] alias isLeast_op_norm := isLeast_opNorm
 
-theorem opNorm_nonneg : 0 ≤ ‖f‖ :=
+theorem opNorm_nonneg (f : ContinuousMultilinearMap 𝕜 E G) : 0 ≤ ‖f‖ :=
   Real.sInf_nonneg fun _ ⟨hx, _⟩ => hx
 
 @[deprecated (since := "2024-02-02")] alias op_norm_nonneg := opNorm_nonneg
 
 /-- The fundamental property of the operator norm of a continuous multilinear map:
 `‖f m‖` is bounded by `‖f‖` times the product of the `‖m i‖`. -/
-theorem le_opNorm : ‖f m‖ ≤ ‖f‖ * ∏ i, ‖m i‖ := f.isLeast_opNorm.1.2 m
+theorem le_opNorm (f : ContinuousMultilinearMap 𝕜 E G) (m : ∀ i, E i) :
+    ‖f m‖ ≤ ‖f‖ * ∏ i, ‖m i‖ :=
+  f.isLeast_opNorm.1.2 m
 
 @[deprecated (since := "2024-02-02")] alias le_op_norm := le_opNorm
 
-variable {f m}
-
-theorem le_mul_prod_of_le_opNorm_of_le {C : ℝ} {b : ι → ℝ} (hC : ‖f‖ ≤ C) (hm : ∀ i, ‖m i‖ ≤ b i) :
+theorem le_mul_prod_of_le_opNorm_of_le {f : ContinuousMultilinearMap 𝕜 E G}
```

Comment:
I've just noticed that this lemma is misnamed (probably, by myself). I'll fix it tomorrow.